tests.py

```diff
@@ -299,6 +341,37 @@ def test_load_ui_into_custom_pyqt4():
         app.exit()
 
 
+def test_load_ui_invalid_class_name():
+    """load_ui: Invalid class name in .ui
+
+    ('Name:', u'MainWindow', 'Class:', u'QMainWindow', 'Parent:', None)
+    ('Name:', u'centralwidget', 'Class:', u'QWidget',
+     'Parent:', <PySide.QtGui.QMainWindow object at 0x1036d3ea8>)
+    ('Name:', u'pushButton', 'Class:', u'QPushButton',
+     'Parent:', <PySide.QtGui.QWidget object at 0x1036d3ea8>)
+    ('Name:', u'menubar', 'Class:', u'QMenuBar',
+     'Parent:', <PySide.QtGui.QMainWindow object at 0x1036d3ea8>)
+    ('Name:', u'statusbar', 'Class:', u'QStatusBar',
+     'Parent:', <PySide.QtGui.QMainWindow object at 0x1036d3ea8>)
+
+    """
+
+    with pyside():
+        from Qt import QtWidgets, load_ui
+
+        class MainWindow(QtWidgets.QMainWindow):
```

Comment:
Do we need this class to test this feature? Looks like a plain load should excersize the same functionality.